### DISCUSSION TITLE
Remote iOS device hanging before Deploy at code signing.

### DISCUSSION THREAD

**Discussion by DclearEE** [unoplatform/uno#9489] [Q&A]
Visual Studio 17.3 stable
Uno (version '1.5.3')
I ran Uno-Check in CMD and got the "everything looks great!"

I started a new Uno Platform project in VS. I tried to remotely build and deploy on a physical iPhone connected to a Mac on my network.
Everything looks great until I get to the code signing part. It get's stuck here, I have waited over 10minutes.. 
The Mac is able to build and sign on it's own via XCode and from a MAUI project via Visual Studio 17.3.
iPhone and Mac are both on and connected.
Picture here: 
![iPhoneDeployHanging](https://user-images.githubusercontent.com/16282199/184405181-ea9f805f-837b-4ccd-931c-071fcb752683.PNG)

For reference I am able to build and deployed a new MAUI project on my iPhone this same way and it is working.  
Am I doing something wrong? Is anyone able to get this to work or is this functionality with iOS still in the works with Uno?

**Comment by jeromelaban** [comment]
Thanks for the report. This is a part of the tooling we do not manage, but when the build stalls this way, it can sometimes mean that there's a popup on the mac asking for the password for the app to be signed.
